src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,137 @@
+        User actual = registrationService.register(user);
+        Assertions.assertEquals(expected, actual);
+    }
+
+    @Test
+    void register_userIsAdult_ok() {
+        user.setAge(DEFAULT_AGE);
+        user.setPassword(DEFAULT_PASSWORD);
+        user.setLogin(DEFAULT_LOGIN);
+        User expected = user;
+        User actual = registrationService.register(user);
+        Assertions.assertEquals(expected, actual);
+    }
+
+    @Test
+    void register_userAgeLess18_notOk() {
+        user.setAge(15);
+        user.setPassword(DEFAULT_PASSWORD);
+        user.setLogin(DEFAULT_LOGIN);
+        Assertions.assertThrows(RuntimeException.class, () ->
+                registrationService.register(user)
+        );
+    }
+
+    @Test
+    void register_userAgeIsNull_notOk() {
+        user.setAge(null);
+        user.setPassword(DEFAULT_PASSWORD);
+        user.setLogin(DEFAULT_LOGIN);
+        Assertions.assertThrows(RuntimeException.class, () ->
+                registrationService.register(user)
+        );
+    }
+
+    @Test
+    void register_userPasswordIsValid_ok() {
+        user.setAge(DEFAULT_AGE);
+        user.setPassword("nvfjkd");
+        user.setLogin(DEFAULT_LOGIN);
+        User expected = user;
+        User actual = registrationService.register(user);
+        Assertions.assertEquals(expected, actual);
+    }
+
+    @Test
+    void register_userPasswordShort_notOk() {
+        user.setAge(DEFAULT_AGE);
+        user.setPassword("hdbf");
+        user.setLogin(DEFAULT_LOGIN);
+        Assertions.assertThrows(RuntimeException.class, () ->
+                registrationService.register(user)
+        );
+    }
+
+    @Test
+    void register_userPasswordIsNull_notOk() {
+        user.setAge(DEFAULT_AGE);
+        user.setPassword(null);
+        user.setLogin(DEFAULT_LOGIN);
+        Assertions.assertThrows(RuntimeException.class, () ->
+                registrationService.register(user)
+        );
+    }
+
+    @Test
+    void register_passwordIs2Null_notOk() {
+        user.setAge(DEFAULT_AGE);
+        user.setPassword(DEFAULT_PASSWORD);
+        user.setLogin(DEFAULT_LOGIN);
+        registrationService.register(user);
+        Assertions.assertThrows(RuntimeException.class, () ->
+                registrationService.register(user)
+        );
+    }
+
+    @Test
+    void getAge_ageIsNull_notOk() {
+        user.setAge(null);
+        Assertions.assertThrows(RuntimeException.class, () ->
+                registrationService.register(user)
+        );
+    }
+
+    @Test
+    void getLogin_loginIsNull_notOk() {
+        user.setLogin(null);
+        Assertions.assertThrows(RuntimeException.class, () ->
+                registrationService.register(user)
+        );
+    }
+
+    @Test
+    void getPassword_passwordIsNull_notOk() {
+        user.setPassword(null);
+        Assertions.assertThrows(RuntimeException.class, () ->
+                registrationService.register(user)
+        );
+    }
```

Comment:
You 
```suggestion
    void getAge_ageIsNull_notOk() {
        user.setAge(null);
        Assertions.assertThrows(RuntimeException.class, () ->
                registrationService.register(user)
        );
    }

    @Test
    void getLogin_loginIsNull_notOk() {
        user.setLogin(null);
        Assertions.assertThrows(RuntimeException.class, () ->
                registrationService.register(user)
        );
    }

    @Test
    void getPassword_passwordIsNull_notOk() {
        user.setPassword(null);
        Assertions.assertThrows(RuntimeException.class, () ->
                registrationService.register(user)
        );
    }
```

Reply:
You don't need to check get() because it's not in RegistrationServiceImpl class, and you already have checked password and age for null.